tilequeue/process.py

```diff
@@ -254,8 +254,8 @@ def lookup_source(source):
         result = Source('osm', source)
     elif source == 'naturalearthdata.com':
         result = Source('ne', source)
-    elif source == 'openstreetmapdata.com':
-        result = Source('shp', source)
+    elif source in ('openstreetmapdata.com', 'osmdata.openstreetmap.de'):
```

Comment:
Why provide allowance for both here?

Reply:
Ah, mostly for CircleCI. Should we instead make this just a test of new value, merge it and then review the main PR?

Reply:
It also helps to guard against using the new code with older RAWR tiles, which have the value baked-in. I think we should probably leave this in until we've done a few builds and remove it before the 1.8.0 release.